.github/workflows/image.yml

```diff
@@ -27,5 +27,5 @@ jobs:
         pull: true
         push: true
         tags: |
-          quay.io/gatekeeper/operator:latest
-          quay.io/gatekeeper/operator:master
+          quay.io/gatekeeper/gatekeeper-operator:latest
+          quay.io/gatekeeper/gatekeeper-operator:master
```

Comment:
@font be sure this registry exists and that the quay token we're using in git has access to push

Reply:
Yes, thanks @tchughesiv !